test/test_combinatorica.py

```diff
@@ -1,11 +1,15 @@
 # -*- coding: utf-8 -*-
-from .helper import evaluate, check_evaluation
+from .helper import evaluate, check_evaluation, reset_session
+
+
+def test_00_initialize():
```

Comment:
This is another thing that I am not sure about how to do. What I want here is that an initializing routine being run just before the tests. In the way that was implemented before, the `evaluate(...)` call was done before all the tests in `mathics-core/test/` be run, so a call to ``session.reset`` unloaded the required package.

Reply:
> This is another thing that I am not sure about how to do. What I want here is that an initializing routine being run just before the tests. In the way that was implemented before, the `evaluate(...)` call was done before all the tests in `mathics-core/test/` be run, so a call to `session.reset` unloaded the required package.

https://stackoverflow.com/questions/22627659/run-code-before-and-after-each-test-in-py-test may help